tests/libtest/thrift_client_to_db.py

```diff
@@ -194,23 +202,26 @@ def get_all_run_results(client, run_id, sort_mode=[], filters=[]):
     return results
 
 
-def get_viewer_client(port, host='localhost', uri='/',
+def get_viewer_client(product, port, host='localhost',
+                      endpoint='/CodeCheckerService',
                       auto_handle_connection=True,
                       session_token=None):
 
-    return CCViewerHelper(host,
-                          port,
-                          uri,
+    return CCViewerHelper(host, port, product,
+                          endpoint,
                           auto_handle_connection,
                           session_token)
 
 
-def get_server_client(port, host='localhost'):
-    return CCReportHelper(host, port)
-
-
 def get_auth_client(port, host='localhost', uri='/Authentication',
                     auto_handle_connection=True, session_token=None):
     return CCAuthHelper(host, port, uri,
                         auto_handle_connection,
                         session_token)
+
+
+def get_product_client(port, host='localhost', uri='/Products',
```

Comment:
Ask the original author of the test suite, the one who wrote the functions above. I kept myself to this scheme.

They have the name `Helper` in their types just as how the Thrift client _helpers_ (call stub forwarders) in `libcodechecker/libclient` do. These objects are _roughly_ the same as those, but they don't expose their method stubs to the environment either, and act as a true low-level forwarders... whereas those in `libclient` have at least a method stub defined.